content/articles/what-happens-when-a-domain-expires.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: What happens when a domain expires?
+excerpt: Description of the different statuses a domain can have after its expiration.
+categories:
+- Domains
+---
+
+# What happens when a domain expires?
+
+When a domain expires, it enters into a grace period. Most registry offer a non-guaranteed 29-day grace period for renewal for many TLDs. Each registry has different rules for their grace periods. There is no guaranteed grace period. 
+
+After a domain has passed through any applicable expiration grace period, the domain may go into redemption status. "Redemption Grace Period" or RGP, is the grace period provided after the registrar has issued a delete request to the registry.
+
+Once the Registry deletes the domain, it becomes available for anyone to purchase as a new domain. It is available on a first-come first-serve basis.
+
+For the WHOIS privacy, there is also a grace period for it. Note that the contact info will be deleted along with the domain. 
```

Comment:
I don't know that I'd include a note about the Whois privacy at all, since it's specific to eNom and not the registries.